betty/extension/http_api_doc/__init__.py

```diff
@@ -2,20 +2,42 @@
 
 from __future__ import annotations
 
+from asyncio import to_thread
 from pathlib import Path
+from shutil import copy
 from typing import TYPE_CHECKING, final
 
+from betty.generate import GenerateSiteEvent
 from typing_extensions import override
 
 from betty.extension.webpack import Webpack, WebpackEntryPointProvider
 from betty.locale.localizable import _, Localizable
 from betty.project.extension import Extension
 
 if TYPE_CHECKING:
+    from betty.event_dispatcher import EventHandlerRegistry
     from betty.machine_name import MachineName
     from collections.abc import Sequence
 
 
+async def _generate_assets(event: GenerateSiteEvent) -> None:
```

Comment:
Gives this a more descriptive name that communicates the one purpose this handler has